spec/unit/cli_spec.rb

```diff
@@ -73,6 +73,24 @@
         allow(SystemDescription).to receive(:delete)
       end
 
+      describe "filter-note" do
```

Comment:
Your'e not really describing `filter-note` here but the behavior of `#inspect`, so I would get rid of that line.